src/vast_tracker.js

```diff
@@ -225,6 +236,13 @@ export class VASTTracker extends EventEmitter {
    * @return {Boolean}
    */
   isQuartileReached(quartile, time, progress) {
```

Comment:
Not sure if it's usefull to add a safeguard here, as `isQuartileReached` is not part of [public methods](https://github.com/dailymotion/vast-client-js/blob/master/docs/api/vast-tracker.md#public-methods--) and its inputs are already covered in `setProgress` method